### PR TITLE
removed te

### PR DESCRIPTION
**Pull request by Hrashi100ms**
Removed old token endpoints and v1 sample app

### CODE REVIEW

File: docs/react-native/v2/quickstart/token-endpoint.mdx
Comment (mntrspace) [inline docs/react-native/v2/quickstart/token-endpoint.mdx:1]: Let's mark it as deprecated and hide the endpoint, but call out that this endpoint has been deprecated. Only show partial endpoint.

File: docs/javascript/v2/how-to-guides/extend-capabilities/plugins/krisp-noise-cancellation.mdx [hunk 1]
```diff
@@ -38,7 +38,11 @@ stop(): void;
 
 ```
 
-Checking if noise cancellation is enabled for the room. Please reach out to us to get this enabled for you.
+Checking if noise cancellation is enabled for the room.
+
+> Note: Prebuilt also supports noise cancellation, to enable it add the import as above and ensure it's enabled from your prebuilt dashboard. From the [dashboard](https://dashboard.100ms.live/), select the template, go to "Customize Prebuilt" -> "Screens and Components" -> "Noise Cancellation State".
+> You will also need to toggle it in the [dashboard's](https://dashboard.100ms.live/)"Template" -> "Advanced Settings" -> "Noise Cancellation"
```
[the remark this answers is not in the document]
Comment (mntrspace) [reply docs/javascript/v2/how-to-guides/extend-capabilities/plugins/krisp-noise-cancellation.mdx:44]: Let's make the above changes for all the other files in Noise Cancellation

File: yarn.lock
Comment (mntrspace) [inline yarn.lock:1]: Why is this file coming in? @Hrashi100ms @KaustubhKumar05

Comment (KaustubhKumar05) [reply yarn.lock:1]: Looks like npm was used instead of yarn. Can you revert changes to this file @Hrashi100ms ?

Comment (KaustubhKumar05) [reply yarn.lock:1]: Was a package-lock.json also generated?

**Pull request closed without merging**